### ISSUE TITLE
Software update - Page down / Page up disabled

### ISSUE THREAD

**Issue by jeepcook** [openatv/enigma2#2160]
Hello I think with the last changes around Software updater, the left and right key doesn't work anymore to do a page up and page down like we did in previous versions.
Probable commits:
7a7b088ce907a50d9674e47bc7a11c87e8d7912c
24a8936b391b469cf44577fea5c8e3b0ce7cbf65
d97f3a6c6883f192f50d7ea67e7becf9a4cc6d8f
9c17f2ba1d50473458cc0ccfe4b54f7fec72044e

Thanks

**Comment by IanSav** [comment]
The LEFT and RIGHT buttons are for horizontal navigation.  When you want to move up and down the lists on OpenATV use the vertical navigation buttons:
REW -> Move to first line
CH+ -> Move up a page/screen
UP -> Move up a line
DOWN -> Move down a line
CH+ -> Move down a page/screen
FF -> Move to last line

The horizontal navigation buttons on OpenATV are:
STOP -> Move to the first option item in a list or character in a text string
LEFT -> Move to the previous option item in a list or character in a text string
RIGHT -> Move to the next option item in a list or character in a text string
PLAY -> Move to the last option item in a list or character in a text string

You should also try HELP as a reminder of options that are available at any point in time.  Help is being added to more and more screens with every update.

**Comment by jeepcook** [comment]
OK so it's CH+ / CH- that doesn't work anymore in the software update screen.

**Comment by jeepcook** [comment]
And the graphic advancement has no interest because it does not represent the real progress of the process

**Comment by IanSav** [comment]
The scrollbar animation was requested by another user who was concerned that there was no feedback that the code was actually running while it was busy fetching data from the Internet.  While some of us have fast Internet connections some people do not.  This can make the UI look like it has frozen.

**Comment by IanSav** [comment]
If there is a button documented in the HELP screen that is not working then please post a bug report and include a copy of the relevant log file.

**Comment by jeepcook** [comment]
I don't understand your answer. Before this ridiculous modifications, the page up/down was working a the software update screen and now doesn't.
So just switch on your box and do the test you will see the problem.
Close this issue if you want, it's not major but don't ask me to open a new one. This one is an issue.

**Comment by jbleyel** [comment]
I cannot reproduce your issue .. CH+/CH- works on my box.

**Comment by 4l3x2k** [comment]
Try with neutrino keymap.

It doesn't work.

**Comment by 4l3x2k** [comment]
@jeepcook 
Is it true that you're using the neutrino keymap?

**Comment by jeepcook** [comment]
No I don't use neutino keymap.. 
But to do the test you have to have more than 10 updates in the software update list, so more than one screen.

**Comment by 4l3x2k** [comment]
Installed 7.0 from 29.12.2021

![image](https://user-images.githubusercontent.com/25264/148273318-1bbf2eb3-d0ac-4344-bb38-fb5bb2a0e196.png)

Press "right"

![image](https://user-images.githubusercontent.com/25264/148273443-f28ca787-1990-4758-8a30-663d0f68bc13.png)

**Comment by jbleyel** [comment]
I have the latest code from GitHub.
.. No Issues.

**Comment by jeepcook** [comment]
So I will retry with a next update containing at least 2 pages of updates.

**Comment by jeepcook** [comment]
Finally, installed the 20201230 image and do a software update, 44 available, right/left are ok to go next/previous screen but in this version the progress bar doesn't exist yet. So not sure it'a the good test to do.

**Comment by jbleyel** [comment]
Image from 20201230 is too old.

**Comment by jeepcook** [comment]
Sorry, I wanted to say 20211230

**Comment by jbleyel** [comment]
Please call 

`opkg list-installed "enigma2"`

in console

**Comment by jeepcook** [comment]
enigma2 - 7.0+git30760+edb76e0-r0

**Comment by jeepcook** [comment]
Hello, why closed ? I will retest the page up/down with future updates.

**Comment by jeepcook** [comment]
Ok sorry I haven't seen the update. I will compile a new version with this fix. Thanks
